### PR TITLE
Allow configurable activation email support link.

### PR DESCRIPTION
**Pull request by UmanShahzad**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [x] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [x] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [x] Add an entry to the CHANGELOG.
  - [ ] Have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)? -- (This link does not work for me; "Page not found")

---

This is a child PR of [#15161 from edx/edx-platform](https://github.com/edx/edx-platform/pull/15161), where we allow the activation email support link to be configurable.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @UmanShahzad! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fconfiguration&number=3886"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 - Role: edxapp
+  - Added the EDXAPP_ACTIVATION_EMAIL_SUPPORT_LINK URL with default value [https://support.edx.org/hc/en-us/articles/227340127-Why-haven-t-I-received-my-activation-email-].
+
```
Comment (jibsheet) [inline CHANGELOG.md]: Edx's own support documentation is not the correct default.
It should either not link anywhere by default, or have a generic help URL.

This URL is something that would be configured on edx's production systems using our internal configuration systems.

Comment (UmanShahzad) [reply CHANGELOG.md]: Ah! So sorry about that.

**Comment by UmanShahzad** [comment]
@jibsheet Addressed the changes -- just waiting on [#15161 from edx/edx-platform](https://github.com/edx/edx-platform/pull/15161) to get through review by maybe 1 more person, or maybe Douglas' review is enough.

**Comment by jibsheet** [comment]
FYI - If you've hardcoded our support URL into the platform I'll have the same complaint.  The defaults are supposed to be useful for openedx deployers, for example we link to different readthedocs documentation.

**Comment by UmanShahzad** [comment]
@jibsheet Hmm, I actually did so [here](https://github.com/edx/edx-platform/pull/15161/files#diff-82bda515879c96ab0641bfcbe58631c3R2271). But this was actually by the request of the reviewer. Is it possible for you to post a comment on there mentioning this, or maybe I can reference this PR there?

**Comment by UmanShahzad** [comment]
P.S. [This was the request](https://github.com/edx/edx-platform/pull/15161#discussion_r118666073).

**Comment by UmanShahzad** [comment]
@jibsheet Thanks for getting back on it. So the more reasonable approach would be to set it to `''` by default but, if it remains that way in any install, it'll default to `SUPPORT_SITE_LINK`.

**Comment by UmanShahzad** [comment]
P.S. [#15161 from edx/edx-platform](https://github.com/edx/edx-platform/pull/15161) is now merged.

**Comment by UmanShahzad** [comment]
@jibsheet Just a reminder ping :+1:

**Review by feanil — APPROVED** [review]

**Review by jibsheet — APPROVED** [review]

**Pull request merged**